Resources/Prototypes/Objectives/thief.yml

```diff
@@ -205,15 +205,16 @@
     difficulty: 1
 
 - type: entity
-  parent: BaseThiefStealObjective
-  id: ClothingHeadsetAltMedicalStealObjective
+  parent: BaseThiefStealCollectionObjective
+  id: ClothingCommandHeadsetStealObjective #imp edit: i'm branching this out from just the CMO's headset to all command headsets so there is less of a blatant kick-me sign on the CMO
```

Comment:
this could probably go in _Impstation: right now it's under the medical subgroup header which feels strange